src/test/java/seedu/duke/classes/WatchTimeTest.java

```diff
@@ -12,7 +12,7 @@
     @Test
     void userReportString() {
         assertEquals("Date : 2020-10-19" + System.lineSeparator()
-                + "Time left today : 60 minutes."
+                + "Time left today : 1 hour(s) 0 minutes."
```

Comment:
Can I just check if our code can parse hours on top of minutes?

Reply:
I just made it so that it prints out hours and minutes. Still cannot parse yet.

Reply:
Ah okay, will raise an issue.